src/devices/wec2103.c

```diff
@@ -0,0 +1,95 @@
+/** @file
+    WEC-2103 temperature/humidity sensor
```

Comment:
Should end with a dot "." and be repeated as first line in the next doc-comment.